### PR TITLE
ZH-trans: add kubeadm_upgrade_node_experimental-control-plane.md

### PR DESCRIPTION
**Pull request by xichengliudui**
ZH-trans: add kubeadm_upgrade_node_experimental-control-plane.md

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_node_experimental-control-plane.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+
+升级部署在此节点上的控制平面实例，重要。在另一个控制平面实例上执行 `kubeadm upgrade apply` 之后，应该执行此命令
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_node_experimental-control-plane.md]:
```suggestion
升级部署在此节点上的控制平面实例。【重要】执行此命令之前，需要在控制平面的另一实例上执行 `kubeadm upgrade apply`。
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_node_experimental-control-plane.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+<!--
+### Synopsis
+-->
+
+### 概要
+
+<!--
+Downloads the kubelet configuration from a ConfigMap of the form "kubelet-config-1.X" in the cluster, where X is the minor version of the kubelet. kubeadm uses the --kubelet-version parameter to determine what the desired kubelet version is. Give
+-->
+从群集中 "kubelet-config-1.X" 的 ConfigMap 下载 kubelet 配置，在集群中 X 是 kubelet 的次要版本。kubeadm 使用 --kubelet-version 参数来确定所需的 kubelet 版本。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_node_experimental-control-plane.md]: 群集 -> 集群

Comment (xichengliudui) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_node_experimental-control-plane.md]: 已完成！

**Pull request merged**